types/jsonwebtoken/tsconfig.json

```diff
@@ -10,7 +10,8 @@
         "strictFunctionTypes": true,
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "exactOptionalPropertyTypes": true
```

Comment:
📝 https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes

Reply:
hey, those are for clients consuming definitions, in essence: typescript projects.
It's not default option, this repo (DT) provides only compatibility with this TS configuration (better worded: promotes, we advice to use it)

Reply:
Hey @peterblazejewicz thanks for the clarification. 
This configuration option needs to ensure correct behavior for type.
I don't see any constraints in the [documentation](https://github.com/DefinitelyTyped/DefinitelyTyped#tsconfigjson) regarding this configuration option change.